### ISSUE TITLE
App just hang up forever without showing any obvious error

### ISSUE THREAD

**Issue by benapetr** [claudehohl/Stikked#488]
This I found in apache error logs:

```
[Tue Apr 03 12:50:41.270879 2018] [:error] [pid 31216] [client 10.130.181.194:52434] PHP Warning:  date(): It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected the timezone 'UTC' for now, but please set date.timezone to select your timezone. in /opt/paste/htdocs/system/core/Log.php on line 185
[Tue Apr 03 12:50:41.270925 2018] [:error] [pid 31216] [client 10.130.181.194:52434] PHP Stack trace:
[Tue Apr 03 12:50:41.270944 2018] [:error] [pid 31216] [client 10.130.181.194:52434] PHP   1. {main}() /opt/paste/htdocs/index.php:0
[Tue Apr 03 12:50:41.270952 2018] [:error] [pid 31216] [client 10.130.181.194:52434] PHP   2. require_once() /opt/paste/htdocs/index.php:315
[Tue Apr 03 12:50:41.270958 2018] [:error] [pid 31216] [client 10.130.181.194:52434] PHP   3. load_class() /opt/paste/htdocs/system/core/CodeIgniter.php:200
[Tue Apr 03 12:50:41.270964 2018] [:error] [pid 31216] [client 10.130.181.194:52434] PHP   4. CI_Hooks->__construct() /opt/paste/htdocs/system/core/Common.php:196
[Tue Apr 03 12:50:41.270970 2018] [:error] [pid 31216] [client 10.130.181.194:52434] PHP   5. load_class() /opt/paste/htdocs/system/core/Hooks.php:90
[Tue Apr 03 12:50:41.270977 2018] [:error] [pid 31216] [client 10.130.181.194:52434] PHP   6. CI_Config->__construct() /opt/paste/htdocs/system/core/Common.php:196
[Tue Apr 03 12:50:41.270983 2018] [:error] [pid 31216] [client 10.130.181.194:52434] PHP   7. log_message() /opt/paste/htdocs/system/core/Config.php:113
[Tue Apr 03 12:50:41.270988 2018] [:error] [pid 31216] [client 10.130.181.194:52434] PHP   8. CI_Log->write_log() /opt/paste/htdocs/system/core/Common.php:469
[Tue Apr 03 12:50:41.270995 2018] [:error] [pid 31216] [client 10.130.181.194:52434] PHP   9. date() /opt/paste/htdocs/system/core/Log.php:185
[Tue Apr 03 12:50:41.271005 2018] [:error] [pid 31216] [client 10.130.181.194:52434] PHP  10. _error_handler() /opt/paste/htdocs/system/core/Log.php:185
[Tue Apr 03 12:50:41.271016 2018] [:error] [pid 31216] [client 10.130.181.194:52434] PHP  11. CI_Exceptions->log_exception() /opt/paste/htdocs/system/core/Common.php:618
[Tue Apr 03 12:50:41.271026 2018] [:error] [pid 31216] [client 10.130.181.194:52434] PHP  12. log_message() /opt/paste/htdocs/system/core/Exceptions.php:105
[Tue Apr 03 12:50:41.271036 2018] [:error] [pid 31216] [client 10.130.181.194:52434] PHP  13. CI_Log->write_log() /opt/paste/htdocs/system/core/Common.php:469
[Tue Apr 03 12:50:41.271045 2018] [:error] [pid 31216] [client 10.130.181.194:52434] PHP  14. date() /opt/paste/htdocs/system/core/Log.php:185

**Comment by benapetr** [comment]
OS: Red Hat Enterprise Linux Server release 7.3 (Maipo)
PHP 5.4.16

**Comment by 6-ril** [comment]
I had the same issue. You just have to add timezone in your php.ini 

https://stackoverflow.com/questions/16765158/date-it-is-not-safe-to-rely-on-the-systems-timezone-settings

**Comment by benapetr** [comment]
ugh, I will check that, isn't there a way to print some user friendly error message instead of locking up?

**Comment by jlpoolen** [comment]
I moved to a newer server and am having the same problem of the screen just hanging with no error messages.  

I do have date.timezone set:
``date.timezone = America/Los_Angeles``
and I'm still experiencing hanging.  Nothing in the Apache logs helps, I have a <Directory> directive:
``LogLevel      trace6``
set and still nothing helpful and the when I try to access ...\paste\inde.php the browser just hangs.

I also confirmed that access to the MySQL database is working using a small test page.  Something is happening at a really low level and there's no error reporting.
